### ISSUE TITLE
Thoughts on contributing an updated version of the React example 

### ISSUE THREAD

**Issue by jagretz** [tastejs/todomvc#2231]
**tl;dr** I'd like to update the current version in the examples for React by creating a version that utilizes the latest syntax and features (ie. context & hooks).

**Background** I was helping a junior developer learn some React and navigated here to show how to build a todo app using React and noticed the current version is outdated and isn't an example I'd want them spending time on. It wraps the components in IIFEs and uses class-based react component instantiation (`componentDidMount`, etc.). I do see a version under `/examples/react-hooks` which at least utilizes hooks, but the dev had trouble installing it with the current version of node & npm. I also noticed that was also 4+ years old. 

I'd like to help contribute an updated version of the React example that utilizes the latest features. I've read through the contribution guidelines and have no problem helping to maintain this example in the years to come.

TodoMVC has helped me to learn unfamiliar tech in the past, and I'd like to pay that forward by providing an up to date example using React. Let me know if this is possible or needs more discussion. Thanks!

**Comment by 17ChristianO** [comment]
I was looking into this issue as part of exploring the repository. It looks like the React example on master has since been updated to use useReducer and functional components with a webpack build (Node 18.13.0+ required). The original concerns raised here seem to have been addressed, so would it make sense to close this issue?

**Comment by raquelpn1979-tech** [comment]
ff28547a368fa83f1cf9625dc68c3cc0c485ede6
